nucypher/config/storages.py

```diff
@@ -476,29 +475,26 @@ def remove(self, checksum_address: str, metadata: bool = True, certificate: bool
 
         if metadata is True:
             metadata_filepath = self.__generate_metadata_filepath(checksum_address=checksum_address)
-            os.remove(metadata_filepath)
+            metadata_filepath.unlink()
             self.log.debug("Deleted {} from the filesystem".format(checksum_address))
 
         if certificate is True:
             certificate_filepath = self.generate_certificate_filepath(checksum_address=checksum_address)
-            os.remove(certificate_filepath)
+            certificate_filepath.unlink()
             self.log.debug("Deleted {} from the filesystem".format(checksum_address))
 
         return
 
     def clear(self, metadata: bool = True, certificates: bool = True) -> None:
         """Forget all stored nodes and certificates"""
 
-        def __destroy_dir_contents(path) -> None:
-            try:
-                paths_to_remove = os.listdir(path)
-            except FileNotFoundError:
+        def __destroy_dir_contents(path: Path) -> None:
+            if not path.is_dir():
                 return
-            else:
-                for file in paths_to_remove:
-                    file_path = os.path.join(path, file)
-                    if os.path.isfile(file_path):
-                        os.unlink(file_path)
+            for file in path.iterdir():
+                file_path = path / file
```

Comment:
I think (?) the objects returned by `iterdir()` are already Path objects to the relevant file...